src/Main.hs

```diff
@@ -1,16 +1,46 @@
-main = putStrLn "It works!"
+import Options.Applicative
 
-wrap :: Int -> String -> [String]
-wrap n s = foldl (splitWordsAt n) [] $ words s
+data Config = Config
+  { width :: Int
+  , path :: FilePath
+  }
+
+config :: Parser Config
+config = Config
+    <$> option auto
+        ( long "width"
+        <> short 'w'
+        <> metavar "WIDTH"
```

Comment:
This is what shows up in the help.

```
Usage: hfold [-w|--width WIDTH] PATH
  Intellegently wrap lines to a given length

Available options:
  -h,--help                Show this help text
  -w,--width WIDTH         Specify a line width to use instead of the default 80
                           columns
  PATH                     Path to input file
```